test/Tests.sh

```diff
@@ -0,0 +1,202 @@
+( ../build/vowpalwabbit/vw -k -l 20 --initial_t 128000 --power_t 1 -d train-sets/0001.dat      -f models/0001_1.model -c --passes 8 --invariant      --ngram 3 --skips 1 --holdout_off ) >outputs/1_Test.stdout.track 2>outputs/1_Test.stderr.track
```

Comment:
Please remove one off testing scripts unless they can be adapted to be generally applicable